src/main/java/org/candlepin/dto/manifest/v1/ConsumerDTO.java

```diff
@@ -33,6 +33,7 @@ public class ConsumerDTO extends CandlepinDTO<ConsumerDTO> {
     protected ConsumerTypeDTO type;
     protected String urlWeb;
     protected String urlApi;
+    protected String regAuthMethod;
```

Comment:
I really dont think we need to be exporting this field to the manifest. This is only valueable for statistics/reporting of auth methods used in Hosted.